tide/core/PDFInteractionDelegate.cpp

```diff
@@ -52,12 +52,12 @@ PDFInteractionDelegate::PDFInteractionDelegate( ContentWindow& contentWindow )
 
 void PDFInteractionDelegate::swipeLeft()
 {
-    _getPDFContent().previousPage();
+    _getPDFContent().nextPage();
```

Comment:
yes, it's funny that we intuitively expect the behaviour to be inverted between PDFs and Webbrowsers :-)